libc/test/include/CMakeLists.txt

```diff
@@ -79,3 +79,207 @@ add_libc_test(
   DEPENDS
     libc.include.llvm-libc-macros.stdckdint_macros
 )
+
+add_libc_test(
+  signbit_test
+  SUITE
+    libc_include_tests
+  SRCS
+    signbit_test.cpp
+  DEPENDS
+    libc.include.llvm-libc-macros.math_function_macros
+  LINK_LIBRARIES
+    LibcFPTestHelpers
```

Comment:
I don't think these tests need anything from `LibcFPTestHelpers`.  Can you remove the `LINK_LIBRARIES` for these tests and see if it's still ok?